### PR TITLE
Add VIA support to Noxary x268

### PR DESCRIPTION
**Pull request by Rozakiin**
## Description
Adds VIA support to the Noxary x268.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/noxary/x268/rules.mk [hunk 1]
```diff
@@ -14,17 +14,17 @@ BOOTLOADER = atmel-dfu
...
 CONSOLE_ENABLE = no        # Console for debug
 COMMAND_ENABLE = no        # Commands for debug and configuration
 # Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
 SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 # if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
-NKRO_ENABLE = yes            # USB Nkey Rollover
-BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality on B7 by default
-RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality on B7 by default
```
Comment (fauxpark) [inline keyboards/noxary/x268/rules.mk:26]:
```suggestion
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
```

File: keyboards/noxary/x268/x268.c [hunk 2]
```diff
@@ -15,35 +15,29 @@
...
 
+// Optional override functions below.
+// You can leave any or all of these undefined.
+// These are only required if you want to perform custom actions.
+
 void matrix_init_kb(void) {
 	// put your keyboard start-up code here
 	// runs once when the firmware starts up
-
+	// setPinOutput(B1);
```
Comment (fauxpark) [inline keyboards/noxary/x268/x268.c:25]:
```suggestion
```

File: keyboards/noxary/x268/keymaps/default/readme.md [hunk 3]
```diff
@@ -1 +1,3 @@
-# The default keymap for x268
+![x268 Layout Image](https://i.imgur.com/D9jQz6g.png)
+
+# The default keymap for x268
```
Comment (fauxpark) [inline keyboards/noxary/x268/keymaps/default/readme.md:3]:
```suggestion
# The default keymap for x268

![x268 Layout Image](https://i.imgur.com/D9jQz6g.png)
```

File: keyboards/noxary/x268/keymaps/via/readme.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+![x268 Layout Image](https://i.imgur.com/D9jQz6g.png)
+
+# The VIA keymap for x268
```
Comment (fauxpark) [inline keyboards/noxary/x268/keymaps/via/readme.md:3]:
```suggestion
# The VIA keymap for x268

![x268 Layout Image](https://i.imgur.com/D9jQz6g.png)
```

File: keyboards/noxary/x268/x268.c [hunk 5]
```diff
@@ -15,35 +15,29 @@
...
+//     return true;
+// }
 
 void led_set_kb(uint8_t usb_led) {
 	// put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
-
 	led_set_user(usb_led);
 }
-
 void led_set_user(uint8_t usb_led) {
```
Comment (fauxpark) [inline keyboards/noxary/x268/x268.c:40]: This function should be updated to `led_update_kb()`

Comment (Rozakiin) [reply keyboards/noxary/x268/x268.c:39]: Updated function

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**